nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/DownloadService.java

```diff
@@ -77,6 +84,14 @@ public long getLength() {
     }
 
     /**
+     * Stores the blobs for downloading it later
+     *
+     * @param the list of blobs to store
```

Comment:
`@param blobs the list of blobs to store`
Also explain the return parameter ("store key") and how it is used (add a `@see` to the methods that take a store key).